index.md

```diff
@@ -1,26 +1,15 @@
-Anbox Cloud provides a rich software stack that enables you to run Android applications in the cloud for all kinds of different use cases, including high-performance streaming of graphics to desktop and mobile client devices.
+Anbox Cloud is a cloud platform that containerises mobile workloads using Android as a guest operating system.
 
-Using container technology, Anbox Cloud is scalable from a single-machine installation that can run scores of single Android systems to an entire cluster of machines. It is based on powerful and battle-proven software from Canonical like [LXD](https://linuxcontainers.org/) and [Juju](https://jujucharms.com/).
+It provides a rich software stack that enables you to run Android applications in the cloud. Anbox Cloud manages the container and application life cycle while optimising for high density, performance and fast container boot times.
 
-## Core features
+Using container technology, Anbox Cloud is scalable from a single-machine installation that can run scores of single Android systems to an entire cluster of machines. Deploying the solution is simple and straightforward, and it is based on powerful and battle-proven software from Canonical like [LXD](https://linuxcontainers.org/) and [Juju](https://jujucharms.com/).
 
-* Simple and straightforward deployment
-* Management of the container and application life cycle while optimising for high density, performance and fast container boot times
-* Platform integration tools to allow, for example, integration of existing streaming solutions
-* Support for both x86 and Arm64 hardware, providing the same set of features
+Anbox Cloud can be used for different kinds of use cases, from delivering central apps from the cloud to high-performance streaming of graphics to desktop and mobile client devices. Its platform integration tools allow, for example, the integration of existing streaming solutions. Anbox Cloud supports x86 and Arm64 hardware, providing the same set of features for both architectures.
```

Comment:
This paragraph also seems to describe what Anbox Cloud does, and belongs with the second paragraph.

Instead, I'd like it to name the users whom it will help. Whose life will be made better by having Anbox Cloud? What industry/kind of organisation/kind of person benefits inn powerful ways?

Reply:
How about:
```suggestion
You should consider using Anbox Cloud if you want to deliver mobile applications that don't depend on the capabilities of your user's devices and that offload compute, storage and energy-intensive applications from the end device to the cloud. Typical use cases include mobile game streaming services, corporate application streaming, application automation and Android application testing.
```

Reply:
+1